plenum/test/view_change/helper.py

```diff
@@ -378,3 +379,10 @@ def nodes_received_ic(nodes, frm, view_no=1):
     for n in nodes:
         assert n.view_changer.instance_changes.has_inst_chng_from(view_no,
                                                                  frm.name)
+
+def check_prepare_certificate(nodes, ppSeqNo):
+    for node in nodes:
+        key = (node.viewNo, ppSeqNo)
+        quorum = node.master_replica.quorums.prepare.value
+        assert node.master_replica.prepares.hasQuorum(ThreePhaseKey(*key),
+                                                       quorum)
```

Comment:
Why don't we check that we have PrePrepare as well?

Reply:
A Prepare is added into `prepares` only if there is a corresponding PrePrepare already received.